tests/testlib/s2n_connection_test_utils.c

```diff
@@ -332,3 +332,12 @@ S2N_RESULT s2n_set_all_mutually_supported_groups(struct s2n_connection *conn)
 
     return S2N_RESULT_OK;
 }
+
+S2N_RESULT s2n_skip_handshake(struct s2n_connection *conn)
```

Comment:
Since this is no longer a static test method, I'd probably change the name to s2n_connection_skip_handshake. Or maybe s2n_connection_set_complete? I definitely think the name could use a revisit ;)